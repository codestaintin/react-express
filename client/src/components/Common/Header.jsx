import React from 'react';
import { Link } from 'react-router-dom';

export const Header = () => (
  <nav className="navbar navbar-expand-lg navbar-dark bg-primary custom_nav">
    <ul className="navbar-nav">
      <li className="nav-item dropdown">
        <Link className="nav-link" to="/">Grocery</Link>
      </li>
    </ul>
  </nav>
);

export default Header;
