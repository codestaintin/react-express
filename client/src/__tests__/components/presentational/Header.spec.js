import React from 'react';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';
import Header from '../../../components/Common/Header';

describe('<Header>', () => {
  let shallowComponent;
  beforeEach(() => {
    shallowComponent = shallow(<Header />);
  });

  it('renders the Header component', () => {
    const tree = toJson(shallowComponent);
    expect(tree).toMatchSnapshot();
  });
});