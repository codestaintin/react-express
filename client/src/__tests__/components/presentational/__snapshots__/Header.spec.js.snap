// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> renders the Header component 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-dark bg-primary custom_nav"
>
  <ul
    className="navbar-nav"
  >
    <li
      className="nav-item dropdown"
    >
      <Link
        className="nav-link"
        replace={false}
        to="/"
      >
        Grocery
      </Link>
    </li>
  </ul>
</nav>
`;
